crates/swc/tests/fixture/deno/paths/issue-2724/output/src/index.ts

```diff
@@ -1,8 +1,8 @@
 "use strict";
-var _interop_require_wildcard = require("@swc/helpers/lib/_interop_require_wildcard.js").default;
+var _interopRequireWildcardMjs = require("../../../../../../../../../packages/swc-helpers/src/_interop_require_wildcard.mjs");
```

Comment:
there is something strange.
swc generated wrong path on my machine. but the correct code  on CI.

I don't know how does the `importResolver` work.

Reply:
![image](https://user-images.githubusercontent.com/7829098/174553403-49746c1f-4335-4e02-bcd6-0a8041f55188.png)
I checked out the main branch, and run into the same issue on my machine(MacBook Pro (13-inch, M1, 2020)).
Do you have any clues? @kdy1

Reply:
Can you try running yarn unlink @swc/helpers?

Reply:
I tried but no luck.
nothing linked to `@swc/helpers`, no global package in yarn.
I cleaned the yarn cache and nothing changed.